### DISCUSSION TITLE
_createnovalidate API error

### DISCUSSION THREAD

**Discussion by ankitbhansali** [egovernments/DIGIT-DevOps#1816] [Q&A]
Currently, All pods are running fine in QUick Setup.
I can access the employee page.
But when I run the Postman collection to create superuser, i get the below response.

"Errors": [
{
"code": "BadRequest",
"message": "400 : [{"ResponseInfo":null,"Errors":[{"code":"InvalidAccessTokenException","message":"An unhandled exception occurred on the server","description":"Invalid Access Token Exception","params":null}]}]",
"description": "400 : [{"ResponseInfo":null,"Errors":[{"code":"InvalidAccessTokenException","message":"An unhandled exception occurred on the server","description":"Invalid Access Token Exception","params":null}]}]",
"params": null
}
]

Can someone please help?

**Comment by talele08** [comment]
Duplicate ticket: https://github.com/egovernments/Digit-Core/discussions/131
